src/components/Filter/FilterInput.vue

```diff
@@ -484,17 +495,21 @@ $input-height: rem(28px);
 
   &__wrapper {
     border: 1px solid var(--input-border-color);
-    background: var(--color-fill);
     border-radius: $small-border-radius;
 
     &--reversed {
       display: flex;
       flex-direction: column-reverse;
     }
+
+    &--no-border-style {
+      border: none;
+    }
   }
 
   &__suggested-tags {
     border-top: 1px solid var(--color-fill-gray-tertiary);
+    background-color: var(--color-fill);
```

Comment:
I could be wrong, but this will add a slight cuttoff to the rounded edges when focused. Notice the top-left edge vs bottom-left. This will be very prominent with larger radii 

![image](https://user-images.githubusercontent.com/9863944/224672648-34cf81c2-3149-40ce-bdcd-fec0730c7386.png)

Reply:
Good catch. I'll need to dig into the earlier changes in the PR to see if this is necessary and fix it if so.

Reply:
It was necessary, but I reverted this to how it was before and applied the bg color override for the quicknav filter explicitly in a safer way that doesn't impact the base behavior.

ca4a3287a1dd9eb634e322504dde8e8c63187c45